File: demo/vb/InheritedTestTests.vb
Imports NUnit.Framework

Namespace NUnitTestDemo

    Public MustInherit Class InheritedTestBaseClass

        <Test>
        Public Sub TestInBaseClass()

        End Sub

    End Class

    Public Class InheritedTestDerivedClass
        Inherits InheritedTestBaseClass

    End Class

End Namespace

File: demo/cpp/InheritedTestTests.cpp
using namespace System;
using namespace NUnit::Framework;

namespace NUnitTestDemo
{
    public ref class InheritedTestBaseClass
    {
	public:
        [Test]
        void TestInBaseClass()
        {
        }
	};

    public ref class InheritedTestDerivedClass : InheritedTestBaseClass
    {
	};
}
